test/index.spec.ts

```diff
@@ -12,4 +12,29 @@ describe('resin-semver', () => {
 			});
 		});
 	});
+
+	describe('.major()', () => {
+		it('should correctly match valid semver versions', () => {
+			expect(semver.major('4.05.1')).to.equal(4);
```

Comment:
This isn't actually a valid semver version (`semver.valid('4.05.1')` returns null)